conf/github-contributions.nginx.conf

```diff
@@ -26,4 +35,14 @@ server {
         alias /github-archive/timeline/;
         autoindex on;
     }
+
+    location ~ /.well-known {
+        allow all;
+    }
```

Comment:
Gotcha. That was the missing piece of the puzzle for me.

On Sun, Feb 7, 2016 at 11:03 AM Liam <notifications@github.com> wrote:

> In conf/github-contributions.nginx.conf
> <https://github.com/tenex/github-contributions/pull/58#discussion_r52119610>
> :
>
> > @@ -26,4 +35,14 @@ server {
> >          alias /github-archive/timeline/;
> >          autoindex on;
> >      }
> > +
> > +    location ~ /.well-known {
> > +        allow all;
> > +    }
>
> Oh and also, the .well-known thing is where the letsencrypt client shoves
> some file so that their verifying server can verify that we still have the
> domain.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tenex/github-contributions/pull/58/files#r52119610>.
>